### PR TITLE
Internal: Remove pages at the end of a Playwright test for Core repo [ED-13110]

### PR DESCRIPTION
**Pull request by hein-obox**
## PR Checklist

- [ ] The commit message follows our guidelines:  https://github.com/elementor/elementor/blob/master/.github/CONTRIBUTING.md

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## Summary

This PR can be summarized in the following changelog entry:

*

## Description
An explanation of what is done in this PR

*

## Test instructions
This PR can be tested by following these steps:

*

## Quality assurance

- [ ] I have tested this code to the best of my abilities
- [ ] I have added unittests to verify the code works as intended
- [ ] Docs have been added / updated (for bug fixes / features)

Fixes #

### CODE REVIEW

**Review by dmaziar — DISMISSED** [review]

File: tests/playwright/sanity/editor-v2.test.ts [hunk 1]
```diff
@@ -28,9 +28,10 @@ test.describe( 'Editor v2', () => {
 
 		await wrapper.getByRole( 'button', { name: 'Post Settings' } ).click();
 
-		await editor.page.getByLabel( 'Title', { exact: true } ).fill( 'Test page' );
+		await editor.page.getByLabel( 'Title', { exact: true } ).fill( 'Playwright Test Page' );
 
-		await wrapper.getByRole( 'button', { name: 'Test page' } ).waitFor();
+		await wrapper.getByRole( 'button', { name: 'Playwright Test Page' } ).waitFor();
+		await editor.page.waitForTimeout( 2000 );
```
Comment (hein-obox) [inline tests/playwright/sanity/editor-v2.test.ts:34]: The timeline at the top of the editor was causing unequal screenshots. I have added this timeout, to make sure that the timeline has disappeared.

Comment (dmaziar) [reply tests/playwright/sanity/editor-v2.test.ts:34]: can we wait for selector not to be visible? We should avoid this hardcoded timeouts.

Comment (hein-obox) [reply tests/playwright/sanity/editor-v2.test.ts:34]: I am not sure where the timeline styling is set. I don't know how to get around this.

@dmaziar

Comment (hein-obox) [reply tests/playwright/sanity/editor-v2.test.ts:34]: Screenshot 2023-12-20 at 07 45 11

It is this pink/purple line at the top.

Comment (hein-obox) [reply tests/playwright/sanity/editor-v2.test.ts]: @dmaziar Updated with the `isUitStable` method.

**Review by dmaziar — APPROVED** [review]

**Pull request merged**